actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -202,15 +202,6 @@ def method_missing(called, *args, &block)
       # HTML5 <tt>data-*</tt> attributes can be set with a single +data+ key
       # pointing to a hash of sub-attributes.
       #
-      # To play nicely with JavaScript conventions sub-attributes are dasherized.
-      # For example, a key +user_id+ would render as <tt>data-user-id</tt> and
-      # thus accessed as <tt>dataset.userId</tt>.
-      #
-      # Values are encoded to JSON, with the exception of strings, symbols and
-      # BigDecimals.
-      # This may come in handy when using jQuery's HTML5-aware <tt>.data()</tt>
-      # from 1.4.3.
-      #
```

Comment:
@kaspth Yes, but these are descriptions of options, and about what options can be passed here. 
We already describe these options before, with exact same desciption.